ports/atmel-samd/boards/matrixportal_m4/mpconfigboard.mk

```diff
@@ -13,6 +13,8 @@ LONGINT_IMPL = MPZ
 CIRCUITPY_LTO_PARTITION = one
 
 CIRCUITPY_AESIO = 0
+CIRCUITPY_FLOPPYIO = 0
```

Comment:
Is this on by default? Do we expect it to work on most boards? Maybe it should be off by default.

Reply:
It is on for all "FULL_BUILD" samd M4 boards (as well as for rp2040).

It should work on any SAM D/E5x board with enough I/Os.

Reply:
Ok, this is fine for now. If "enough I/Os" are only available on a few boards then we should consider making it opt-in.